backend/cybnetics/endpoints.py

```diff
@@ -1,8 +1,118 @@
-from flask import Flask, jsonify
+import json
+import ast
+import jwt
+import datetime
+
+from flask import Flask, jsonify, request, session
 
 app = Flask(__name__)
 app.config.from_pyfile('./config.py')
 
-@app.route('/test', methods=['GET'])
-def test():
-    return jsonify({'hello': 'world'})
+
+db = app.config['DATABASE']
+
+# This is just a sanity check from myself
+@app.route("/")
+def get_initial_response():
+    """Welcome message for the API."""
+    # Message to the user
+    message = {
+        'apiVersion': 'v1.0',
+        'status': '200',
```

Comment:
Api version in the response is more useful for larger scale applications, but for one of this size we don't really need to return it. also we don't need to return the status code because its already part of the http response